sources/web/datalab/static/datalab.css

```diff
@@ -526,7 +521,7 @@ div.modal-dialog pre, div.modal-dialog code, #help code, #help pre {
 }
 
 /* Responsive Design */
-@media screen and (max-width: 1270px) {
+@media screen and (max-width: 1230px) {
```

Comment:
if we're removing all the buttons on the right side of the toolbar, i think we can afford less than this. i'd keep about 50px on top of the `maintoolbar`'s width and that should be enough.